environment/db-migrate.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
This script will be used in Codebuild to run migration in e2e-test, preprod, and prod environments.